webapp/pages/settings/index.vue

```diff
@@ -101,7 +101,7 @@ export default {
     async submit() {
       this.loadingData = true
       const { name, about } = this.formData
-      let { locationName } = this.formData
+      let { locationName } = this.formData || this.currentUser
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-09T13:44:09Z" title="Monday, September 9th 2019, 3:44:09 pm +02:00">Sep 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -101,7 +101,7 @@ export default {
     async submit() {
       this.loadingData = true
       const { name, about } = this.formData
-      let { locationName } = this.formData
+      let { locationName } = this.formData || this.currentUser
```


Although this is already changing the behaviour that you want to debug.